.dockerignore

```diff
@@ -0,0 +1,10 @@
+out
```

Comment:
I added this file so that when testing cifuzz locally you don't make giant docker images that contain `out`.